sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero_queue.cpp

```diff
@@ -1358,6 +1358,10 @@ ur_result_t ur_queue_handle_t_::synchronize() {
     if (ImmCmdList == Queue->CommandListMap.end())
       return UR_RESULT_SUCCESS;
 
+#if (L0_USE_ZECOMMANDLISTHOSTSYNCHRONIZE == 1)
```

Comment:
Why would this be a build-time decision not run-time? Can\t we check the version of the loaded L0 and run one code or another?

Reply:
this was to bypass while we upgrade the GPU driver, but I see now there's a way to do that. We need 

ci-neo-master-026152: Update to L0 Loader v1.10.0 or higher. 

Current PR for uplift in https://github.com/intel/llvm/pull/10087